### PR TITLE
Publish deb and rpm packages to Bintray

### PR DESCRIPTION
**Pull request by ob-stripe**
### Reviewers
r? @tomer-stripe @brandur-stripe 
cc @stripe/dev-platform

 ### Summary
Publish the `.deb` and `.rpm` packages created by GoReleaser to repositories on Bintray:
- https://bintray.com/stripe/stripe-cli-deb
- https://bintray.com/stripe/stripe-cli-rpm

Users of Debian-based and RedHat-based distributions will be able to use these repositories to keep the CLI up-to-date.

The publish scripts were adapted from this repo: https://github.com/leopardslab/dunner. I've run them locally and verified that they work (after making a couple of changes, notably telling curl to follow redirects when downloading the `.deb`/`.rpm` file from GitHub).

I've also set the `BINTRAY_USER` and `BINTRAY_API_KEY` env vars in Travis.

### CODE REVIEW

**Comment by ob-stripe** [comment]
Installation instructions for users, to add to the README once we've verified this works correctly:

## Debian-based distributions

For the first time use, add Bintray's GPG Key and add stripe-cli to the apt repository list:

```
sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net --recv-keys 379CE192D401AB61
sudo echo "deb https://dl.bintray.com/stripe/stripe-cli-deb stable main" | tee -a /etc/apt/sources.list
```

Then install the CLI with:

```
sudo apt-get update
sudo apt-get install stripe
```

## RedHat-based distributions

For the first time use, add stripe-cli to the repository list:

```
wget https://bintray.com/stripe/stripe-cli-rpm/rpm -O bintray-stripe-stripe-cli-rpm.repo
sudo mv bintray-stripe-stripe-cli-rpm.repo /etc/yum.repos.d/
```

Then install the CLI with:

```
sudo dnf update
sudo dnf install stripe
```

File: scripts/publish-deb-to-bintray.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    if [ "$(echo $RESPONSE_CODE | head -c2)" != "20" ]; then
+        echo "Unable to put in download list, HTTP response code: $RESPONSE_CODE"
+        exit 1
+    fi
+    echo "HTTP response code: $RESPONSE_CODE"
+  done
+}
+
+snooze () {
+    echo "\nSleeping for 30 seconds. Have a coffee..."
```
Comment (tomer-stripe) [inline scripts/publish-deb-to-bintray.sh]: Reading this at 6:30am, this really spoke to me 😆

**Comment by brandur-stripe** [comment]
Very cool!!

Would you mind putting some notes at the top of the scripts that just have a link back to the repo they were adapted from? I'm kind of hopeful that might be somewhat helpful to retrieve a new version if they ever break somewhere down the line. (I don't trust shell scripts.) Even if they don't, it's still a nice annotation to help indicate that they're not entirely homegrown.

**Comment by jcanaves-trv** [comment]
Thankyou, saved my day

**Pull request merged**